lib/util/error_handling/error_details.dart

```diff
@@ -0,0 +1,83 @@
+/*
+* Orion - Error Details
+* Copyright (C) 2024 TheContrappostoShop
+*
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+class ErrorDetails {
+  final String title;
+  final String message;
+
+  ErrorDetails(this.title, this.message);
+}
+
+final Map<String, ErrorDetails> errorLookupTable = {
+  'default': ErrorDetails(
+    'Unknown Error',
+    'An unknown error has occurred. Please contact support.',
+  ),
+  'PINK-CARROT': ErrorDetails(
```

Comment:
There's a PINK-CARROT usage elsewhere in the code--should that be linked to this?

Reply:
I don't think I'm following, could you please elaborate?

Reply:
Nevermind, I was misremembering what I had read farther up in the code